pkg/agent/agent.go

```diff
@@ -206,17 +210,17 @@ func (i *Initializer) Initialize() error {
 	}
 
 	// Install Openflow entries on OVS bridge
-	if err := i.initOpenFlowPipeline(); err != nil {
+	if err := i.initOpenFlowPipeline(getRoundNum(i.ovsBridgeClient)); err != nil {
```

Comment:
It relies on a variable of initializer, I think typically it should be its method(and logically it can be)? I think @jianjuns did the refactor to unify the methods and functions.

Reply:
Thanks, I removed the parameter and let the `getRoundNum` be invoked in `initOpenFlowPipeline`.